test/e2e/json-rpc/eth_accounts.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('eth_accounts', function () {
           .withKeyringControllerAdditionalAccountVault()
           .withPreferencesControllerAdditionalAccountIdentities()
           .withAccountsControllerAdditionalAccountIdentities()
-          .withPermissionControllerConnectedToTestDapp()
+          .withPermissionControllerConnectedToTestDappWithTwoAccounts()
```

Comment:
How did this work before 🤔

Reply:
there are 3 permitted accounts in `withPermissionControllerConnectedToTestDapp`, but only 2 exist on the keychain so only 2 get returned

Reply:
Ah I see, makes sense